src/core/MIGRATION.md

```diff
@@ -1170,7 +1170,6 @@ import { setup, start } from '../core_plugins/visualizations/public/legacy';
 | `import 'ui/query_bar'`                           | `import { QueryStringInput } from '../data/public'`          | Directives are deprecated.                                                                                                           |
 | `import 'ui/search_bar'`                          | `import { SearchBar } from '../data/public'`                 | Directive is deprecated.                                                                                                             |
 | `import 'ui/kbn_top_nav'`                         | `import { TopNavMenu } from '../navigation/public'`          | Directive is still available in `ui/kbn_top_nav`.                                                                                    |
-| `ui/saved_objects/components/saved_object_finder` | `import { SavedObjectFinder } from '../kibana_react/public'` |                                                                                                                                      |
```

Comment:
Why was this removed? Shouldn't it just change the location to the new plugin?

Reply:
I updated the path. Thanks.